src/README.md

```diff
@@ -13,6 +11,8 @@ Select FlightSQL from the Type dropdown.
 
 Find out more here:
 
-https://github.com/influxdata/grafana-flightsql-datasource#grafana-flight-sql-datasource
+[Github](https://github.com/influxdata/grafana-flightsql-datasource#grafana-flight-sql-datasource)
```

Comment:
This is how these render on github: https://github.com/influxdata/grafana-flightsql-datasource/tree/chore/remove-comment/src

![Screenshot 2023-04-24 at 3 51 47 PM](https://user-images.githubusercontent.com/490673/234101295-72d7fc9c-f7e8-42ba-bd9c-e09fec43b500.png)


Which seems like a nice improvement to me 👍